src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -414,6 +414,11 @@
     "DESCRIPTION": "Gain !M! gold. NL Remember stsjorbsmod:Charity. NL Exhaust.",
     "UPGRADE_DESCRIPTION": "Gain !M! gold. NL Remember stsjorbsmod:Charity. NL Gain stsjorbsmod:Clarity of current memory. NL Exhaust."
   },
+  "stsjorbsmod:Overkill": {
+    "NAME": "Overkill",
+    "DESCRIPTION": "Deal damage equal to overkill damage to all other enemies.",
```

Comment:
The way this is worded made me think it was going to be dealing damage immediately; maybe instead, `Whenever you kill an enemy, deal any overkill damage to ALL enemies.`?

Reply:
(similar feedback for power description)